setup.py

```diff
@@ -85,6 +85,7 @@ def run(self):
         )
         self._run("Running black…", [sys.executable, "-m", "black", f"{here}/slack"])
         self._run("Running flake8…", [sys.executable, "-m", "flake8", f"{here}/slack"])
+        self._run("Running flake8…", [sys.executable, "-m", "bandit", "-r", f"{here}/slack"])
```

Comment:
Let's update the comment from `flake8` to `bandit`.

Reply:
Apologies, missed that, will do.

Reply:
Thanks for catching that.

Reply:
@RodneyU215 I have fixed it, please have a look at the commit.